boost-maven/boost-runtimes/runtime-openliberty/src/main/java/boost/runtimes/openliberty/boosters/LibertyMPJWTBoosterConfig.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package boost.runtimes.openliberty.boosters;
+
+import static boost.common.config.ConfigConstants.MPJWT_11;
+
+import java.util.Map;
+import boost.common.BoostException;
+import boost.common.BoostLoggerI;
+import boost.common.boosters.MPJWTBoosterConfig;
+import boost.runtimes.openliberty.LibertyServerConfigGenerator;
+import boost.runtimes.openliberty.boosters.LibertyBoosterI;
+
+public class LibertyMPJWTBoosterConfig extends MPJWTBoosterConfig implements LibertyBoosterI {
+	
+  public LibertyMPJWTBoosterConfig(Map<String, String> dependencies, BoostLoggerI logger) throws BoostException {
+	       super(dependencies, logger);
+  }
+		
+  @Override
+  public String getFeature() {        
+	  if (getVersion().equals(MP_20_VERSION)) {
```

Comment:
indentation.. are you running the formatter precommit?

Reply:
Forgot about that.. Let me re-format.